auditbeat/docs/getting-started.asciidoc

```diff
@@ -1,182 +1,70 @@
-
-["source","sh",subs="attributes"]
-------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-amd64.deb
-sudo dpkg -i {beatname_lc}-{version}-amd64.deb
-------------------------------------------------
-
-endif::[]
-
-[[rpm]]
-*rpm:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-x86_64.rpm
-sudo rpm -vi {beatname_lc}-{version}-x86_64.rpm
-------------------------------------------------
-
-endif::[]
-
-[[mac]]
-*mac:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-darwin-x86_64.tar.gz
-tar xzvf {beatname_lc}-{version}-darwin-x86_64.tar.gz
-------------------------------------------------
-
-endif::[]
-
-include::{libbeat-dir}/shared-brew-install.asciidoc[]
-
-[[linux]]
-*linux:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-linux-x86_64.tar.gz
-tar xzvf {beatname_lc}-{version}-linux-x86_64.tar.gz
-------------------------------------------------
-
-endif::[]
-
-[[docker]]
-*docker:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of {beatname_uc} has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes"]
-------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-linux-x86_64.tar.gz
-tar xzvf {beatname_lc}-{version}-linux-x86_64.tar.gz
-------------------------------------------------
-
-endif::[]
-
-See <<running-on-docker, Running on Docker>> for deploying Docker containers.
-
-[[win]]
-*win:*
+[role="screenshot"]
+image::./images/auditbeat-auditd-dashboard.png[{beatname_uc} Auditd dashboard]
 
-ifeval::["{release-state}"=="unreleased"]
+[float]
+=== Before you begin
 
-Version {version} of {beatname_uc} has not yet been released.
+You need {es} for storing and searching your data, and {kib} for visualizing and
+managing it.
 
-endif::[]
+include::{libbeat-dir}/tab-widgets/spinup-stack-widget.asciidoc[]
 
-ifeval::["{release-state}"!="unreleased"]
+[float]
+[[install]]
```

Comment:
not sure how asciidoctor handles these anchors. But as we introduce new labels here, would it make sense to introduce a more complete naming schema? e.g. `install` is a quite generic name.

Reply:
Hmm...that's a good point. In most cases, I just kept the anchor names the same to keep from having to fix links. I think it would actually be pretty useful to have a naming convention. I'm going to defer this for now, but going forward (for new docs) we should come up with a consistent naming pattern for anchors. Naming is especially important in situations where the anchors become the html filename (not here).

Reply:
Created an issue to track this: https://github.com/elastic/beats/issues/19229